README.md

```diff
@@ -276,6 +309,55 @@ datadogWebpackPlugin({
 
 </details>
 
+### Rum <img src="packages/assets/src/esbuild.svg" alt="ESBuild" width="17" /> <img src="packages/assets/src/rollup.svg" alt="Rollup" width="17" /> <img src="packages/assets/src/rspack.svg" alt="Rspack" width="17" /> <img src="packages/assets/src/vite.svg" alt="Vite" width="17" /> <img src="packages/assets/src/webpack.svg" alt="Webpack" width="17" />
+
+> Interact with Real User Monitoring (RUM) directly from your build system.
+
+#### [📝 Full documentation ➡️](/packages/plugins/rum#readme)
+
+<details>
+
+<summary>Configuration</summary>
+
+```typescript
+datadogWebpackPlugin({
+    rum?: {
+        disabled?: boolean,
+        sdk?: {
```

Comment:
Maybe we could use the RUM type directly here, no?

Reply:
That's a very good idea, I'll check if I can.